node/status_node.go

```diff
@@ -404,6 +406,49 @@ func (n *StatusNode) StatusService() (st *status.Service, err error) {
 	return
 }
 
+// DebugService exposes reference to debug service running on top of the node.
+func (n *StatusNode) DebugService() (st *debug.Service, err error) {
+	n.mu.RLock()
+	defer n.mu.RUnlock()
+
+	err = n.gethService(&st)
+	if err == node.ErrServiceUnknown {
+		err = ErrServiceUnknown
+	}
+
+	return
+}
+
+// shhextService exposes reference to sshext service running on top of the node.
+func (n *StatusNode) shhextService() (st *shhext.Service, err error) {
+	n.mu.RLock()
+	defer n.mu.RUnlock()
+
+	err = n.gethService(&st)
+	if err == node.ErrServiceUnknown {
+		err = ErrServiceUnknown
+	}
+
+	return
+}
+
+// ShhextServiceAPI exposes reference to shhext service api.
+func (n *StatusNode) ShhextServiceAPI() (api *shhext.PublicAPI, err error) {
+	st, err := n.shhextService()
+	if err != nil {
+		return
+	}
+
+	apis := st.APIs()
+	if len(apis) == 0 {
+		err = ErrServiceUnknown
+		return
+	}
+
+	api = apis[0].Service.(*shhext.PublicAPI)
```

Comment:
i now think it is just better to add postconfirm to shhext because it logically belongs there and will not introduce such helpers

Reply:
As you guys prefer. I think it makes sense to have the less open endpoints the better, especially if we're using them internally as it will give us the flexibility to change them without worrying about third-party integrations.

We can also explore how to "disable" specific endpoints independently, so they're not based on a specific namespace. That will give us the best of both worlds, ability to set some methods for internal use and keep the code clean under its expected namespace. 

Not sure how to do that, so it will need some more research.

Reply:
Yup, if we can blacklist the specific endpoint it's probably cleaner.